.changeset/rich-badgers-repair.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+Replaced @esm-bundle/chai with chai 5.x which now ships ES modules out of the box
```

Comment:
We usually don't even add descriptive text for empty changesets as these are changes that won't affect users, won't bump package version, and don't add a changelog entry. I don't think having text would cause any troubles though.